src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -657,3 +657,19 @@ function Test-SetAzureStorageAccountStorageV2
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Test GetAzureStorageUsage with current Location
+.Description
+AzureAutomationTest
+#>
+function Test-GetAzureStorageLocationUsage
+{
+        # Test
+        $loc = Get-ProviderLocation_Stage ResourceManagement; 
```

Comment:
@blueww would you mind reformatting this test so it has the proper resource construction/deconstruction? This would be similar to the tests above where we have the following format:

```
# Setup test variables
try
{
    # Create Resources
    # Test logic / assertions
}
finally
{
    # Destroy resources
}
```

Reply:
@cormacpayne 

For this case, it won't create any resource, so also don't need to delete any resource.
It will only get the current storage usage (how many account created, and how many can be create). So I think it don't need the format of resource construction/deconstruction.